include/tgr_3d.xml.h

```diff
@@ -91,6 +91,22 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #define SYNCPT_CONDITIONAL_OP_DONE				0x00000001
 #define SYNCPT_CONDITIONAL_RD_DONE				0x00000002
 #define SYNCPT_CONDITIONAL_WR_SAFE				0x00000003
+#define STENCIL_OP_ZERO						0x00000000
+#define STENCIL_OP_KEEP						0x00000001
+#define STENCIL_OP_REPLACE					0x00000002
+#define STENCIL_OP_INCR						0x00000003
+#define STENCIL_OP_DECR						0x00000004
+#define STENCIL_OP_INVERT					0x00000005
+#define STENCIL_OP_INCR_WRAP					0x00000006
+#define STENCIL_OP_DECR_WRAP					0x00000007
+#define STENCIL_FUNC_NEVER					0x00000000
+#define STENCIL_FUNC_LESS					0x00000001
+#define STENCIL_FUNC_EQUAL					0x00000002
+#define STENCIL_FUNC_LESS_EQUAL					0x00000003
+#define STENCIL_FUNC_GREATER					0x00000004
+#define STENCIL_FUNC_NOTEQUAL					0x00000005
+#define STENCIL_FUNC_GREATER_EQUAL				0x00000006
+#define STENCIL_FUNC_ALWAYS					0x00000007
```

Comment:
Hmm, looks a bit like we should namespace this, seem kinda risky for collisions (and inconsistent with other things here)